src/Maestro/Client/src/MaestroApiOptions.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+
+namespace Microsoft.DotNet.Maestro.Client
+{
+    partial class MaestroApiOptions
+    {
+        private MaestroApiTokenCredential maestroApiTokenCredential;
+        public MaestroApiOptions(MaestroApiTokenCredential maestroApiTokenCredential)
+        {
+            this.maestroApiTokenCredential = maestroApiTokenCredential;
+        }
+        partial void InitializeOptions()
+        {
+            if (Credentials != null)
+            {
+                AddPolicy(new BearerTokenAuthenticationPolicy
+                    (Credentials, Array.Empty<string>()), HttpPipelinePosition.PerCall);
```

Comment:
This is an... unusual wrapping style. I don't think I've ever seen the opening "(" not be attached to the method call/constructor that it's part of.  I'd personally write  it like:
```csharp
  AddPolicy(
    new BearerTokenAuthenticationPolicy(Credentials, Array.Empty<string>()),
    HttpPipelinePositition.PerCall
  );
```

Reply:
changed it